### PR TITLE
.Net: Ishu fixed: Net: Deserializing exception for WithHuggingFaceTextEmbeddingGenerationService #2999

### PR DESCRIPTION
**Pull request by ashishwebt**
### Motivation and Context
  1. Why is this change required?
  This PR fix issue 2999.
  2. What problem does it solve?
  It solves Json parse bug with hugging face text embedding generation service.  
  3. If it fixes an open issue, please link to the issue here.
  https://github.com/microsoft/semantic-kernel/issues/2999

### Description
In this commit, we addressed a bug related to JSON parsing for text embeddings. The previous code used JsonSerializer to deserialize the response into a TextEmbeddingResponse object and then selected the Embedding property from each item. However, this approach was causing JSON parsing errors.

### Contribution Checklist

- [ ] The code builds clean without any errors or warnings
- [ ] The PR follows the [SK Contribution Guidelines](https://github.com/microsoft/semantic-kernel/blob/main/CONTRIBUTING.md) and the [pre-submission formatting script](https://github.com/microsoft/semantic-kernel/blob/main/CONTRIBUTING.md#development-scripts) raises no violations
- [ ] All unit tests pass, and I have added new tests where possible
- [ ] I didn't break anyone :smile:

### CODE REVIEW

File: dotnet/samples/KernelSyntaxExamples/Example60_HuggingFaceEmbedding.cs [hunk 1]
```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.SemanticKernel.AI.Embeddings;
+using Microsoft.SemanticKernel;
+
+public static class Example60_HuggingFaceEmbedding
```
Comment (dmytrostruk) [inline dotnet/samples/KernelSyntaxExamples/Example60_HuggingFaceEmbedding.cs:8]: We already have `Example20_HuggingFace` file with HuggingFace examples, so we can place all new examples related to HuggingFace there.

**Review by dmytrostruk — COMMENTED** [review]
@ashishwebt Thank you for contribution! I left one comment.
Also, please make sure new changes don't introduce new warnings and/or typos. If you will have a chance to resolve them, that would be great! Thanks a lot!

**Comment by SergeyMenshykh** [comment]
@ashishwebt Please remove the `TextEmbeddingResponse` class if it's not used anymore.

**Comment by afederici75** [comment]
Since this is stale, created https://github.com/microsoft/semantic-kernel/pull/3598

**Comment by markwallace-microsoft** [comment]
Closing as obsolete. @RogerBarreto is updating the Hugging Face connector during the current sprint and will improving the robustness of the services.

**Pull request closed without merging**